src/renderers/WebGLRenderer.js

```diff
@@ -1359,33 +1517,37 @@ function WebGLRenderer( parameters ) {
 
 		}
 
-		const programAttributes = program.getAttributes();
+		if ( material.morphTargets || material.morphNormals ) {
```

Comment:
Most of the code from here down is actually part of #19745, but I don't have a good way of filtering that out given that the PR hasn't been merged and the branches I'm doing the PRs against are in my own fork.